packages/medusa/src/api/routes/admin/inventory-items/index.ts

```diff
@@ -222,6 +222,38 @@ export type AdminInventoryItemsListRes = PaginatedResponse & {
   inventory_items: InventoryItemDTO[]
 }
 
+/**
+ * @schema DecoratedInventoryItemDTO
+ * type: object
+ * allOf:
+ *   - $ref: "#/components/schemas/InventoryItemDTO"
+ *   - type: object
+ *     required:
+ *       - stocked_quantity
+ *       - reserved_quantity
+ *     properties:
+ *       location_levels:
+ *         type: array
+ *         items:
+ *           $ref: "#/components/schemas/InventoryLevelDTO"
+ *       variants:
+ *         type: array
+ *         items:
+ *           $ref: "#/components/schemas/ProductVariant"
+ *       stocked_quantity:
+ *         type: number
+ *         description: The total quantity of the item in stock across levels
+ *       reserved_quantity:
+ *         type: number
+ *         description: The total quantity of the item available across levels
+ */
+export type DecoratedInventoryItemDTO = InventoryItemDTO & {
```

Comment:
suggestion: should this move to the types package? Maybe not now since there is a coupling with the variant but the idea would be to have a variant dto as well, we should get rid of returning entity type as it forces us to be structured around the ORM